backend/hdf5/DataArrayHDF5.cpp

```diff
@@ -131,13 +131,13 @@ vector<double> DataArrayHDF5::polynomCoefficients() const {
 }
 
 
-void DataArrayHDF5::polynomCoefficients(const vector<double> &coefficients) {
+void DataArrayHDF5::polynomCoefficients(const vector<double> &coefficients, bool compression) {
     DataSet ds;
     if (group().hasData("polynom_coefficients")) {
         ds = group().openData("polynom_coefficients");
         ds.setExtent({coefficients.size()});
     } else {
-        ds = group().createData("polynom_coefficients", H5T_NATIVE_DOUBLE, {coefficients.size()});
+         ds = group().createData("polynom_coefficients", H5T_NATIVE_DOUBLE, {coefficients.size()}, {}, {}, true, true, compression);
```

Comment:
Another one?

Reply:
eh, fixed those ones, will rebase the branch once the other #696 is merged...